### ISSUE TITLE
admin->connection extra value dont show up

### ISSUE THREAD

**Issue by dimberman** [apache/airflow#7971]
**Apache Airflow version**: None

Ticket was created 08/Dec/16 20:56

**Kubernetes version (if you are using kubernetes)** (use `kubectl version`):

**Environment**:

- **Cloud provider or hardware configuration**:
- **OS** (e.g. from /etc/os-release):
- **Kernel** (e.g. `uname -a`):
- **Install tools**:
- **Others**:
**What happened**:

steps to recreate

	go to http://localhost:8080/admin/connection/
	create new connection and add this into extra field 

 {"verify": False} 

	save the connection, and come back to UI on list of connection
	click on + to see the extra field

It will not show up, but this json string will work: 

 {"verify": "False"} 

basically it does not appear when the value field of the json string is not quoted.
looks like it throws an error 

 {models.py:594} ERROR - No JSON object could be decoded 

in anycase, extra string gets stored into database, but it creates problems when passing the extra params to the external library

**What you expected to happen**:

**How to reproduce it**:

**Anything else we need to know**:

Moved here from https://issues.apache.org/jira/browse/AIRFLOW-688
